responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -504,13 +521,16 @@ def save(self, path):
         dtypes = self.train.dtypes.astype(str).to_dict()
         self._write_to_file(data_directory / (_TRAIN + _DTYPES),
                             json.dumps(dtypes))
+        reset_index_train = self._reset_data_frame_index(self.train)
         self._write_to_file(data_directory / (_TRAIN + _JSON_EXTENSION),
-                            self.train.to_json())
+                            reset_index_train.to_json())
+
         dtypes = self.test.dtypes.astype(str).to_dict()
         self._write_to_file(data_directory / (_TEST + _DTYPES),
                             json.dumps(dtypes))
+        reset_index_test = self._reset_data_frame_index(self.test)
```

Comment:
don't we then need to set the index back when calling load?  I don't quite understand where we undo the changes to reset index?

Reply:
As responded in the previous comment, we are not dropping the index but resetting it.